report/4.Solution/BuiltInFunctions.tex

```diff
@@ -0,0 +1,58 @@
+\subsection{Built-in Functions}
+HCL features several built-in functions that map the languages functionality directly over to its c++ equivalent. 
```

Comment:
Instead of "features" can we use "includes"
Can we also remove "over" as it is redundant, over